subsys/net/lib/wifi_mgmt_ext/wifi_mgmt_ext.c

```diff
@@ -143,7 +143,7 @@ static int add_network_from_credentials_struct_personal(struct wifi_credentials_
 		return -ENOEXEC;
 	}
 
-	LOG_INF("Connection requested");
+	LOG_DBG("Connection requested");
```

Comment:
Was this change intentinal here? If so, I'd put it in a separate commit, it's not really related to the sample.

Reply:
I didn't intent to include this change, ill remove it.